### ISSUE TITLE
MergeTags not working with post meta

### ISSUE THREAD

**Issue by AndyVkBlin** [wpninjas/ninja-forms#3178]
- **What actions are you taking?**
I'm adding to recipient field ('TO') post_meta field like: {post_meta:META KEY}
  - What had happened was... 
The recipient was cleared as NF_MergeTags_WP::setup_post_meta method inside NF_MergeTags_WP::method is never invoked. is_admin functions always returns true during AJAX requests, and $_GET['page'] is also empty.
- **What are you expecting to occur?**
NF_MergeTags_WP::setup_post_meta should be invoked and populated with  post_meta during AJAX requests

- **What is occurring instead?**
Recipient email was cleared inside NF_MergeTags_WP::replace as NF_MergeTags_WP::post_meta property was empty (setup_post_meta was never invoked).

So basically the problem is with this line: https://github.com/wpninjas/ninja-forms/blob/3c74a4d1fc032414bb9c329297704f332a9f6dcb/includes/MergeTags/WP.php#L34
